lambdas/data-migration2/package.json

```diff
@@ -1,48 +1,48 @@
 {
-  "name": "@cumulus/data-migration2",
-  "version": "13.2.1",
-  "description": "A Lambda function used for doing data migrations intended to be executed after data-migration1.",
-  "license": "Apache-2.0",
-  "engines": {
-    "node": ">=14.19.1"
-  },
-  "private": true,
-  "scripts": {
-    "clean": "rm -rf dist",
-    "build-lambda-zip": "cd dist/webpack && node ../../../../bin/zip.js lambda.zip index.js",
-    "package": "npm run clean && npm run prepare && npm run webpack && npm run build-lambda-zip",
-    "test": "../../node_modules/.bin/ava",
-    "test:coverage": "../../node_modules/.bin/nyc npm test",
-    "prepare": "npm run tsc",
-    "tsc": "../../node_modules/.bin/tsc",
-    "tsc:listEmittedFiles": "../../node_modules/.bin/tsc --listEmittedFiles",
-    "webpack": "../../node_modules/.bin/webpack"
-  },
-  "ava": {
-    "files": [
-      "tests/**/*.js"
-    ],
-    "timeout": "15m"
-  },
-  "dependencies": {
-    "@aws-sdk/client-dynamodb": "^3.58.0",
-    "@aws-sdk/lib-dynamodb": "^3.58.0",
-    "@aws-sdk/util-dynamodb": "^3.58.0",
-    "@cumulus/api": "13.2.1",
-    "@cumulus/aws-client": "13.2.1",
-    "@cumulus/common": "13.2.1",
-    "@cumulus/db": "13.2.1",
-    "@cumulus/errors": "13.2.1",
-    "@cumulus/logger": "13.2.1",
-    "@cumulus/message": "13.2.1",
-    "@cumulus/types": "13.2.1",
-    "JSONStream": "1.3.5",
-    "knex": "0.95.15",
-    "lodash": "^4.17.21",
-    "p-map": "^4.0.0",
-    "pg": "^8.3.0"
-  },
-  "devDependencies": {
-    "@cumulus/test-data": "13.2.1"
-  }
-}
\ No newline at end of file
+    "name": "@cumulus/data-migration2",
```

Comment:
This doesn't seem right.   Is it this way on the feature branch?

Reply:
This doesn't appear to be formatted like this on main *or* feature.

Reply:
That would be my new workstation not set up for JSON 🤦 

https://github.com/nasa/cumulus/pull/3181/commits/eca0ea19288236316e9c7cd9407765bf485d3f67